src/Model/Item.php

```diff
@@ -1328,23 +1284,15 @@ private static function tagDeliver($uid, $item_id)
 
 		$forum_mode = (($prvgroup) ? 2 : 1);
 
-		q("UPDATE `item` SET `wall` = 1, `origin` = 1, `forum_mode` = %d, `owner-name` = '%s', `owner-link` = '%s', `owner-avatar` = '%s',
-			`private` = %d, `allow_cid` = '%s', `allow_gid` = '%s', `deny_cid` = '%s', `deny_gid` = '%s'  WHERE `id` = %d",
-			intval($forum_mode),
-			dbesc($c[0]['name']),
-			dbesc($c[0]['url']),
-			dbesc($c[0]['thumb']),
-			intval($private),
-			dbesc($u[0]['allow_cid']),
-			dbesc($u[0]['allow_gid']),
-			dbesc($u[0]['deny_cid']),
-			dbesc($u[0]['deny_gid']),
-			intval($item_id)
-		);
+		$fields = ['wall' => true, 'origin' => true, 'forum_mode' => $forum_mode,
+			'owner-name' => $c[0]['name'], 'owner-link' => $c[0]['url'], 'owner-avatar' => $c[0]['thumb'],
```

Comment:
Can you please replace the query by a `dba::selectFirst` call in line 1274 : https://github.com/friendica/friendica/pull/4487/files#diff-043fd70296cbf595b3caed1b2dc79248R1274 and use a meaningful variable name instead of `$c`?

Reply:
I focussed on the ```insert``` function and only changed the write statements on other places. There are currently 15 calls of ```q(...)``` left in that class. I wouldn't want to just change that single one and left the other untouched.

So it is up to you: Should I complete the database task? If yes, then it will last until tomorrow, since it is too late to start working at it now.

Reply:
Yes, this isn’t an urgent fix or even a release blocker, so I’d rather it was done thoroughly.

Reply:
Okay. I was unsure, if I should do it completely or not. So I just moved the stuff that I already did in that local branch and hadn't touched the other places. I already prepared for some question like "why don't you focus on the last bugs?" :-)